pkg/driver/controller.go

```diff
@@ -222,7 +261,11 @@ func (cs *controller) ListVolumes(
 func validateCapabilities(caps []*csi.VolumeCapability) bool {
 
 	for _, cap := range caps {
-		if !IsSupportedVolumeCapabilityAccessMode(cap.AccessMode.Mode) {
+		if !IsSupportedVolumeCapabilityAccessMode(cap.GetAccessMode().Mode) {
+			return false
+		}
+
+		if cap.GetMount() == nil {
```

Comment:
This check will need to be removed once block device is supported, lets check in a better way

Reply:
done